notebooks/official/model_evaluation/automl_text_classification_model_evaluation.ipynb

```diff
@@ -0,0 +1,1706 @@
+{
```

Comment:
<p>Get predictions for batch prediction job</p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/959/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>